components/home/integrationsHead.tsx

```diff
@@ -15,7 +15,7 @@ export default function IntegrationsHead({
         <title>{name} Integrations - Best Data App Integration Tools</title>
         <meta
           name="description"
-          content={`The Grouparoo ${name} integration allows you to sync all of your customer data that lives in various sources into ${name}. ✓ Start a free trial now!`}
+          content={`Grouparoo\'s ${name} integration allows you to sync all of your customer data that lives in various sources into ${name}. ✓ Start a free trial now!`}
```

Comment:
@tealjulia The apostrophe shows up as below: 
<img width="589" alt="Screen Shot 2021-07-06 at 4 10 40 PM" src="https://user-images.githubusercontent.com/633226/124660881-ceadc780-de74-11eb-92b3-d16fdbde29a5.png">

Evan and I spent a bit of time looking into it and decided the copy change of "The Grouparoo ${name}" gets the job done without the escaping challenges. Thoughts on reverting this change?

Reply:
Yeah no problem